src/components/Search/Homepage/StyledPopper.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Popper } from '@material-ui/core';
```

Comment:
nit: It's better to import the component with the full path, that reduces the bundle size

```tsx
import Popper from '@material-ui/core/Popper;
```